test/objects.js

```diff
@@ -4,6 +4,7 @@
   QUnit.module('Objects');
 
   var testElement = typeof document === 'object' ? document.createElement('div') : void 0;
+  var testNodeList = typeof document === 'object' ? document.querySelectorAll('div') : void 0;
```

Comment:
`querySelector` isn't the best choice as it won't be available in older browsers. Try `document.getElementsByTagName`